CHANGELOG.md

```
@@ -1,52 +1,35 @@
 # [1.1.0](https://github.com/VilnaCRM-Org/website/compare/v1.0.0...v1.1.0) (2024-09-18)
 
-
 ### Bug Fixes
 
-* **#1:** infinite pnpm install ([#41](https://github.com/VilnaCRM-Org/website/issues/41)) ([23c5c18](https://github.com/VilnaCRM-Org/website/commit/23c5c18f0e7647fe2bd4b82a8c63fb131b69264d)), closes [#1](https://github.com/VilnaCRM-Org/website/issues/1) [#1](https://github.com/VilnaCRM-Org/website/issues/1)
-* **#37:** add changelog.md ([2fd6c57](https://github.com/VilnaCRM-Org/website/commit/2fd6c576a04c26a32520cc10fb15d256bfa9d500)), closes [#37](https://github.com/VilnaCRM-Org/website/issues/37)
-* **#37:** update version in autorelease.yml ([2aa93e0](https://github.com/VilnaCRM-Org/website/commit/2aa93e0b643abc7ed4568097aa29faf37bbbad63)), closes [#37](https://github.com/VilnaCRM-Org/website/issues/37)
-
+- **#1:** infinite pnpm install ([#41](https://github.com/VilnaCRM-Org/website/issues/41)) ([23c5c18](https://github.com/VilnaCRM-Org/website/commit/23c5c18f0e7647fe2bd4b82a8c63fb131b69264d)), closes [#1](https://github.com/VilnaCRM-Org/website/issues/1) [#1](https://github.com/VilnaCRM-Org/website/issues/1)
+- **#37:** add changelog.md ([2fd6c57](https://github.com/VilnaCRM-Org/website/commit/2fd6c576a04c26a32520cc10fb15d256bfa9d500)), closes [#37](https://github.com/VilnaCRM-Org/website/issues/37)
+- **#37:** update version in autorelease.yml ([2aa93e0](https://github.com/VilnaCRM-Org/website/commit/2aa93e0b643abc7ed4568097aa29faf37bbbad63)), closes [#37](https://github.com/VilnaCRM-Org/website/issues/37)
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 100; Actual: 210

(MD013, line-length)